code/modules/mob/living/carbon/xenomorph/mutators/strains/crusher/charger.dm

```diff
@@ -16,8 +16,8 @@
 
 /datum/xeno_mutator/charger
 	name = "STRAIN: Crusher - Charger"
-	description = "Your charge is now momentum based, the further you go, the more damage and speed you will get until you achieve max momentum when you roar. Your armor is now directional, being the toughest on the front, weaker on the sides and weakest at the back. Your shield is also removed. In return you gain more health, gain an ability to tumble to avoid enemies, and gain an ability to forcefully move enemies via ramming. Finally, you trade being able to resist slowdowns from tall autospitters for being unaffected by frenzy pheros."
-	flavor_description = "We're just getting started. Nothing stops this train."
+	description = "In exchange for your shield, a little bit of your armor and damage, your slowdown resist from autospitters, your ability to sense frenzy pheromone, and your ability to lock your direction, you gain a considerable amount of health, some speed, and your charge can now be manually-controlled and is now momentum-based; the further you go, the more damage and speed you will gain until you achieve maximum momentum, indicated by your roar. In addition, your armor is now directional, being the toughest on the front, weaker on the sides and weakest at the back. In return, you gain an ability to tumble to avoid enemies and an ability to forcefully move enemies via ramming."
```

Comment:
```suggestion
	description = "In exchange for your shield, a little bit of your armor and damage, your slowdown resist from autospitters, your ability to sense frenzy pheromone, and your ability to lock your direction, you gain a considerable amount of health, some speed, and your charge can now be manually-controlled and is momentum-based; the further you go, the more damage and speed you will gain until you achieve maximum momentum, indicated by your roar. In addition, your armor is now directional, being the toughest on the front, weaker on the sides and weakest at the back. In return, you gain an ability to tumble to avoid enemies and an ability to forcefully move enemies via ramming."
```

Reply:
Fixed, ty!